src/components/DataVisualization/LinearBar/LinearBar.vue

```diff
@@ -1,6 +1,12 @@
 <template>
-  <div class="linear-progress-bar">
-    <div class="progress">
+  <div
+    class="linear-progress-bar"
+    :class="{ 'horizontal-value': horizontalValue }"
+  >
+    <div
+      class="progress"
+      :style="`width: ${barWidth}%; margin-bottom: 0;`"
```

Comment:
wouldn't it be better if u pass a second paramenter called width type to allow the consumer to choose the width type between pixels (px), percent (%) or viewport width (vw)?